pkg/ingester/ingester_v2.go

```diff
@@ -452,3 +461,63 @@ func (i *Ingester) closeAllTSDB() {
 	i.userStatesMtx.Unlock()
 	wg.Wait()
 }
+
+// openExistingTSDB walks the user tsdb dir, and opens a tsdb for each user. This may start a WAL replay, so we limit the number of
+// concurrently opening TSDB.
+func (i *Ingester) openExistingTSDB(ctx context.Context) error {
```

Comment:
I would add a Info log at the beginning and one once completed.

Reply:
Fixed